play/src/main/scala/com/managementapp/authentication/models/UserTable.scala

```diff
@@ -0,0 +1,15 @@
+package com.managementapp.authentication.models
+
+import slick.lifted.Tag
+import slick.driver.MySQLDriver.api._
+
+
+case class User(email: String, password: String)
```

Comment:
Again, about when should we use protobuf. Since protobuf has generated a DTO in Java, can we directly use the generated one (`AccountDTO` class in this case) instead of creating a scala version of DTO again?